### PR TITLE
update: Add references to Observability Accelerator

### PR DESCRIPTION
**Pull request by Howlla**
#### Which issue(s) this PR fixes: 830

Fixes # Multiple references to observability accelerator and the addons we have for CDK. 
Added a reference to kubecost secure ingress blog

#### Quality checks

- [x] My content adheres to the style guidelines
- [x] I ran `make test module="<module>"` it was successful (see https://github.com/aws-samples/eks-workshop-v2/blob/main/docs/automated_tests.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by Howlla** [comment]
@niallthomson @StevenDavid @jsgoudar Please review

**Review by StevenDavid — APPROVED** [review]
This changes look good to me

**Review by StevenDavid — APPROVED** [review]
Looks good to me.

File: website/docs/observability/index.md [hunk 1]
```diff
@@ -20,3 +20,7 @@ Watch a video walk-through of the observability module with one of the module ma
 :::info
 To dive deeper into AWS Observability features take a look at the [One Observability Workshop](https://catalog.workshops.aws/observability/en-US)
 :::
+
+:::info
+Explore a set of opinionated Infrastructure as Code (IaC) modules to help you set up observability for your AWS environments in our [AWS Observability Accelerator for CDK](https://aws-observability.github.io/cdk-aws-observability-accelerator/) and [AWS Observability Accelerator for Terraform](https://aws-observability.github.io/terraform-aws-observability-accelerator/). These modules work with AWS Native services like Amazon CloudWatch and AWS-managed observability services such as Amazon Managed Service for Prometheus, Amazon Managed Grafana and AWS Distro for OpenTelemetry (ADOT).
```
Comment (jsgoudar) [inline website/docs/observability/index.md:25]: should we add the diagram of - Single EKS Cluster Open Source Observability Accelerator, or share some example patterns like - cost monitoring, alerting rules etc, to get people interested and click on the links to learn more ?

**Review by jsgoudar — APPROVED** [review]
Changes look good, one comment if we want to add a little more details in the first page

File: website/docs/observability/open-source-metrics/index.md [hunk 2]
```diff
@@ -28,3 +28,11 @@ In this lab, we'll collect the metrics from the application using [AWS Distro fo
...
+
+:::info
+If you are using CDK Observability Accelerator, checkout the collection of Open Source Observability Patterns covering a wide range of use cases including [ADOT collector](https://aws-observability.github.io/cdk-aws-observability-accelerator/patterns/existing-eks-observability-accelerators/existing-eks-adotmetrics-collection-observability/), [GPU Monitoring with Nvidia DCGM](https://aws-observability.github.io/cdk-aws-observability-accelerator/patterns/single-new-eks-observability-accelerators/single-new-eks-gpu-opensource-observability/) and many more.
+:::
+
+:::tip
+After completing this module checkout how to integrate [ADOT collector and Amazon Managed Prometheus EKS on AWS Fargate Clusters](https://aws.amazon.com/blogs/mt/observability-on-eks-fargate-cluster-with-amazon-managed-service-for-prometheus-and-amazon-managed-grafana/).
```
Comment (niallthomson) [inline website/docs/observability/open-source-metrics/index.md:37]: Does "ADOT collector and Amazon Managed Prometheus EKS on AWS Fargate Clusters" make sense? Whats "Amazon Managed Prometheus EKS"? Also "AWS Fargate Clusters" should be "AWS Fargate clusters"

**Review by niallthomson — CHANGES_REQUESTED** [review]
There are a few places where this is written:

> If you are using CDK Observability Accelerator, checkout [...]

It should be:

> If you are using the CDK Observability Accelerator then check out [...]

Also there are several places that have "Addon" in the middle of a sentence, which should probably be "add-on" consistently.

File: website/docs/observability/index.md (hunk 1 above)
Comment (Howlla) [reply website/docs/observability/index.md:25]: I agree @jsgoudar, adding links to a few of the patterns.

File: website/docs/observability/open-source-metrics/index.md (hunk 2 above)
Comment (Howlla) [reply website/docs/observability/open-source-metrics/index.md:37]: removed.

**Comment by Howlla** [comment]
Hi @jsgoudar and @niallthomson 
I have incorporated the feedback shared in the last review. Please let me know if this meets the bar.

**Pull request merged**